### PR TITLE
Display allocations when allocation ID is not specified

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by nak3** [comment]
Output example is below. It outputs only `running` status.
~~~
$ nomad alloc-status 
ID        Eval ID   Name              Node ID   Job ID   Client Status
0159ce7d  5e9cbc4b  example.cache[2]  542febd4  example  running
aa59171f  5e9cbc4b  example.cache[1]  542febd4  example  running
bd20def2  5e9cbc4b  example.cache[3]  542febd4  example  running
d7bb2083  5e9cbc4b  example.cache[4]  542febd4  example  running
0340ff57  5e9cbc4b  example.cache[0]  542febd4  example  running
~~~

File: command/alloc_status.go [hunk 1]
```diff
@@ -86,6 +85,41 @@ func (c *AllocStatusCommand) Run(args []string) int {
...
+		// No output if we have no jobs
+		if len(allocs) == 0 {
+			c.Ui.Output("No allocations exist")
+			return 0
+		}
+
+		out := make([]string, 0)
+		out = append(out, "ID|Eval ID|Name|Node ID|Job ID|Desired Status|Client Status")
+		for _, alloc := range allocs {
+			// Dislpay only running status
```
Comment (dadgar) [inline command/alloc_status.go]: Seems a bit odd to do this filtering by default with no escape hatch.

Comment (nak3) [inline command/alloc_status.go]: Thank you. Yes, it was unnecessary filtering. I removed it.

**Comment by dadgar** [comment]
I will leave this open for now in case you have a very compelling reason but I think we should actually close this PR. While useful for a small cluster, issuing this command on a large cluster running tens of thousands of allocations would have an extremely bad UX and will not be helpful

**Comment by nak3** [comment]
Yes, you are correct. For a large cluster, the output would be too much. Please close this, since I don't have any reason to convince.

**Comment by dadgar** [comment]
Awesome thanks for all the PRs

**Pull request closed without merging**